examples/MQ/pixelDetector/src/FairOnlineSink.h

```diff
@@ -24,14 +24,13 @@
 
 class FairEventHeader;
 class FairMQRunDevice;
-class TObject;
-class TTree;
 
 class FairOnlineSink : public FairSink
 {
   public:
     FairOnlineSink();
     ~FairOnlineSink() override = default;
+    FairOnlineSink& operator=(const FairOnlineSink&) = delete;
```

Comment:
What speaks against `= default` here?

Reply:
The current copy constructor is private only. So for the public API, this class is not copyable in any way.

I guess FairSinks are meant as "resources" like a mutex.

CloneSink is the only internal user of the copy constructor. CloneSink is meant for multithreading and expected to do strange things (which is another topic in itself).

Reply:
I added a comment to the copy constructor:
```cpp
/// internal helper function for CloneSink()
```

I hope this makes it more obvious?